client/my-sites/site-settings/date-time-format/date-time-format-options.jsx

```diff
@@ -0,0 +1,338 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { includes, startsWith } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ExternalLink from 'components/external-link';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormInput from 'components/forms/form-text-input';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormSelect from 'components/forms/form-select';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import SectionHeader from 'components/section-header';
+import { phpToMomentDatetimeFormat } from 'lib/formatting';
+import wrapSettingsForm from '../wrap-settings-form';
+
+/**
+ * Module constants
+ */
+const defaultDateFormats = [ 'F j, Y', 'Y-m-d', 'm/d/Y', 'd/m/Y' ];
+const defaultTimeFormats = [ 'g:i a', 'g:i A', 'H:i' ];
+
+export class DateTimeFormatOptions extends Component {
+	state = {
+		customDateFormat: false,
+		customTimeFormat: false,
+		isLoadingSettings: true,
+	};
+
+	isDefaultFormat = ( defaultFormats, format ) => ! includes( defaultFormats, format );
```

Comment:
seems fishy if we're not doing anything but calling a standard function. could this be better inlined or further abstracted?

if it's used just once or twice, maybe inlining would be good

if it's used in multiple places, maybe abstracting it out could be helpful…

```js
isDefaultFormat = format => ! includes( defaultFormats, format );
```